sdk/provisioning/Azure.Provisioning.CloudMachine/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ## 1.0.0-beta.2 (Unreleased)
```

Comment:
Nit: need to remove this block, I believe ... or .. are we releasing beta.3 without releasing beta.2?

Reply:
If we haven't release a beta.2 then the correct thing to do undo the version increment here and in the csproj. Only update the version after we do a release.